### PR TITLE
DSS-5178 DSS-5277: Implements Compliance and Confidential Spec

### PR DESCRIPTION
**Pull request by theseyi**
Adds 'logs/' to ignored files

Updates EmberSelectorComponent to handle a list of string options or list of options with value and label, flags the currently selected option, and bubble change actions with 'selectionDidChange' action

DSS-5178: Removes previous updates to search.js: moving jQuery + DOM heavy imperative implementation to Ember component

DSS-5178: Adds templates and components DropRegion and DraggableItem

DSS-5178: Adds getSecuritySpec action and compliance types to Dataset controller, cleans up Datasets route and removes inline securitySpec fetch from route

DSS-5178: Updates templates for compliance spec

DSS-5178: Adds compliance component and updates template

Adds .DS_Store to gitignore

DSS-5277: Adds dataset-confidential component to DOM, Creates DatasetConfidential component, refactors out data handling from component

DSS-5277: Moves data fetching to Dataset Route model and set model data on controller, Adds template for confidential spec component

DSS-5178: Moves view related complianceTypes to component

DSS-5277 DSS-5178: Adds styling for tab content

### CODE REVIEW

**Comment by ericsun2** [comment]
Awesome.
Ship it.

File: web/app/views/index.scala.html [hunk 1]
```diff
@@ -699,10 +700,20 @@
             {{#dataset-access hasAccess=hasAccess accessibilities=accessibilities}}
             {{/dataset-access}}
           </div>
-          <div id="compliancetab" class="tab-pane">
-            {{#dataset-compliance hasCompliance=hasCompliance compliance=compliance}}
-            {{/dataset-compliance}}
-          </div>
+            <div id="compliancetab" class="tab-pane">
+                {{#dataset-compliance securitySpec=securitySpec
```
Comment (luisah) [inline web/app/views/index.scala.html]: It seems you don't need to use the block form for this component. Same for other components throughout this pull request.

File: web/public/javascripts/routers/datasets.js [hunk 2]
```diff
@@ -201,43 +200,62 @@ App.DatasetsRoute = Ember.Route.extend({
...
+    }
+
+    controller.set('datasetId', id);
+    if (params.dataset.schema || params.schema) {
+      let {schema} = params.dataset || params;
+      controller.set('datasetSchemaFieldsAndTypes', getFieldNamesAndTypesFrom(schema));
+      controller.set('securitySpec', params.securitySpec);
+    }
+
+    var instanceUrl = 'api/v1/datasets/' + id + "/instances";
```
Comment (luisah) [inline web/public/javascripts/routers/datasets.js]: var instead of let? and please move to top of block

Comment (theseyi) [inline web/public/javascripts/routers/datasets.js]: Sure, this was actually old code. My change here was line formatting, that changed the indentation for line 258.

**Pull request merged**